diff --git a/CMakeLists.txt b/CMakeLists.txt
index 938d498..a30d0dc 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,7 +1,7 @@
 # Keystone Assembler Engine (www.keystone-engine.org)
 # By Nguyen Anh Quynh, 2016
 
-cmake_minimum_required(VERSION 2.8.7)
+cmake_minimum_required(VERSION 3.5...4.5)
 project(keystone)
 
 set(KEYSTONE_VERSION_MAJOR 0)
@@ -25,7 +25,7 @@ if (POLICY CMP0051)
   # stripped everywhere that access the SOURCES property, so we just
   # defer to the OLD behavior of not including generator expressions
   # in the output for now.
-  cmake_policy(SET CMP0051 OLD)
+  cmake_policy(SET CMP0051 NEW)
 endif()
 
 if (POLICY CMP0063)
diff --git a/kstool/CMakeLists.txt b/kstool/CMakeLists.txt
index d28da48..9b389b8 100644
--- a/kstool/CMakeLists.txt
+++ b/kstool/CMakeLists.txt
@@ -1,7 +1,7 @@
 # Kstool for Keystone assembler engine.
 # By Nguyen Anh Quynh, 2016
 
-cmake_minimum_required(VERSION 2.8)
+cmake_minimum_required(VERSION 3.5...4.5)
 
 project(kstool)
 
diff --git a/llvm/CMakeLists.txt b/llvm/CMakeLists.txt
index d926b08..3d639c4 100644
--- a/llvm/CMakeLists.txt
+++ b/llvm/CMakeLists.txt
@@ -1,6 +1,6 @@
 # See docs/CMake.html for instructions about how to build LLVM with CMake.
 
-cmake_minimum_required(VERSION 2.8.7)
+cmake_minimum_required(VERSION 3.5...4.5)
 
 set(LLVM_INSTALL_TOOLCHAIN_ONLY ON)
 
@@ -19,7 +19,7 @@ if (POLICY CMP0051)
   # stripped everywhere that access the SOURCES property, so we just
   # defer to the OLD behavior of not including generator expressions
   # in the output for now.
-  cmake_policy(SET CMP0051 OLD)
+  cmake_policy(SET CMP0051 NEW)
 endif()
 
 if(CMAKE_VERSION VERSION_LESS 3.1.20141117)
@@ -335,18 +335,22 @@ include(HandleLLVMOptions)
 # Verify that we can find a Python 2 interpreter.  Python 3 is unsupported.
 # FIXME: We should support systems with only Python 3, but that requires work
 # on LLDB.
-set(Python_ADDITIONAL_VERSIONS 2.7)
-include(FindPythonInterp)
-if( NOT PYTHONINTERP_FOUND )
-  message(FATAL_ERROR
-"Unable to find Python interpreter, required for builds and testing.
-
-Please install Python or specify the PYTHON_EXECUTABLE CMake variable.")
-endif()
-
-if( ${PYTHON_VERSION_STRING} VERSION_LESS 2.7 )
-  message(FATAL_ERROR "Python 2.7 or newer is required")
-endif()
+# SCT
+#set(Python_ADDITIONAL_VERSIONS 2.7)
+#include(FindPythonInterp)
+#if( NOT PYTHONINTERP_FOUND )
+#  message(FATAL_ERROR
+#"Unable to find Python interpreter, required for builds and testing.
+#
+#Please install Python or specify the PYTHON_EXECUTABLE CMake variable.")
+#endif()
+#
+#if( ${PYTHON_VERSION_STRING} VERSION_LESS 2.7 )
+#  message(FATAL_ERROR "Python 2.7 or newer is required")
+#endif()
+set(PYTHONINTERP_FOUND TRUE)
+#set(PYTHON_EXECUTABLE "python3") 
+#set(PYTHON_VERSION_STRING "3.14.3") # Or your specific version
 
 ######
 # LLVMBuild Integration
diff --git a/llvm/include/llvm/ADT/STLExtras.h b/llvm/include/llvm/ADT/STLExtras.h
index 3346de9..33ddf88 100644
--- a/llvm/include/llvm/ADT/STLExtras.h
+++ b/llvm/include/llvm/ADT/STLExtras.h
@@ -18,6 +18,7 @@
 #define LLVM_ADT_STLEXTRAS_H
 
 #include "llvm/Support/Compiler.h"
+#include <cstdint>
 #include <algorithm> // for std::all_of
 #include <cassert>
 #include <cstddef> // for std::size_t
diff --git a/samples/CMakeLists.txt b/samples/CMakeLists.txt
index a018ef4..2d26bd4 100644
--- a/samples/CMakeLists.txt
+++ b/samples/CMakeLists.txt
@@ -5,7 +5,7 @@
 # For Windows, be sure to copy keystone.dll or keystone.lib
 # to this directory before compiling
 
-cmake_minimum_required(VERSION 2.8)
+cmake_minimum_required(VERSION 3.5...4.5)
 
 project(sample)
 
